Workflows/ExperimentAcquisitionVR.bonsai

```diff
@@ -1,1267 +1,1730 @@
-              <rx:Name>OmniceptGlia</rx:Name>
-            </Expression>
-            <Expression xsi:type="rx:GroupBy">
-              <rx:KeySelector>Item1</rx:KeySelector>
-              <rx:ElementSelector>Item2</rx:ElementSelector>
-            </Expression>
-            <Expression xsi:type="IncludeWorkflow" Path="Extensions\ZeroMQLogging.bonsai">
-              <LogName>Glia</LogName>
-              <StartIndex>0</StartIndex>
-              <ExpectedIndices>3</ExpectedIndices>
-            </Expression>
-            <Expression xsi:type="rx:BehaviorSubject" TypeArguments="sys:Tuple(sys:String,p2:NetMQMessage)">
-              <rx:Name>VRTransform</rx:Name>
-            </Expression>
-            <Expression xsi:type="rx:GroupBy">
-              <rx:KeySelector>Item1</rx:KeySelector>
-              <rx:ElementSelector>Item2</rx:ElementSelector>
-            </Expression>
-            <Expression xsi:type="IncludeWorkflow" Path="Extensions\ZeroMQLogging.bonsai">
-              <LogName>VRTransform</LogName>
-              <StartIndex>0</StartIndex>
-              <ExpectedIndices>3</ExpectedIndices>
-            </Expression>
-            <Expression xsi:type="rx:BehaviorSubject" TypeArguments="sys:Tuple(sys:String,p2:NetMQMessage)">
-              <rx:Name>UnityRecorderStatus</rx:Name>
-            </Expression>
-            <Expression xsi:type="rx:GroupBy">
-              <rx:KeySelector>Item1</rx:KeySelector>
-              <rx:ElementSelector>Item2</rx:ElementSelector>
-            </Expression>
-            <Expression xsi:type="IncludeWorkflow" Path="Extensions\ZeroMQLogging.bonsai">
-              <LogName>UnityRecorder</LogName>
-              <StartIndex>0</StartIndex>
-              <ExpectedIndices>3</ExpectedIndices>
-            </Expression>
-            <Expression xsi:type="rx:BehaviorSubject" TypeArguments="sys:Tuple(p3:IplImage,sys:Tuple(sys:String,p2:NetMQMessage))">
-              <rx:Name>VRImage</rx:Name>
-            </Expression>
-            <Expression xsi:type="MemberSelector">
-              <Selector>Item1</Selector>
-            </Expression>
             <Expression xsi:type="GroupWorkflow">
-              <Name>LogVideo</Name>
+              <Name>ExperimentMetadata</Name>
+              <Description>Specifies mandatory metadata for the collected dataset.</Description>
               <Workflow>
                 <Nodes>
-                  <Expression xsi:type="WorkflowInput">
-                    <Name>Source1</Name>
+                  <Expression xsi:type="ExternalizedMapping">
+                    <Property Name="Value" DisplayName="DataPath" />
                   </Expression>
-                  <Expression xsi:type="SubscribeSubject">
-                    <Name>LoggingPath</Name>
+                  <Expression xsi:type="PropertySource" TypeArguments="io:GetFiles,sys:String">
+                    <MemberName>Path</MemberName>
+                    <Value>C:\IndoorData</Value>
```

Comment:
Remove absolute path

Reply:
This is the base path for saving data on the main workflow. 
If we remove it it makes mandatory that after clone and before running acquisition workflow user needs to change the path property, and manually set the data folder. 
To ease the process of installation and running we consider having a base folder for data in the c:\ to be useful. 
If user wants can always change it afterwards because is a public property on the `ExperimentMetadata` group in the main workflow.